### ISSUE TITLE
Dynamic Domains

### ISSUE THREAD

**Issue by mikefox** [tehplague/swiftmailer-mailgun-bundle#14]
Currently Domain is a config parameter but depending your use case you might be managing multiple domains with Mailgun in the same application. To achieve this you would need the domain parameter to be configurable at runtime before the Mail is sent.

**Comment by mikefox** [comment]
Could be achieved crudely by adding a setter to the transport class:

```
    public function setDomain($domain)
    {
        $this->domain = $domain;
    }
```

**Comment by exuperok** [comment]
+1

**Comment by Nyholm** [comment]
This issue will be fixed by #16

**Comment by Nyholm** [comment]
Fixed by #16
